packages/@okta/vuepress-site/docs/concepts/session/index.md

```diff
@@ -0,0 +1,51 @@
+
+#### Privileged access use case
+
+For privileged access apps, such as an e-commerce payment app, securing sensitive user information is paramount and application sessions need to be tightly restricted. Application sessions are usually short lived and inactivity timeouts are in the minutes for these types of apps. Developers need to employ best practices and end sessions when they are no longer required. If the application session is terminated, the user must be redirected to the IdP to re-authenticate back into the app for a new session. If there is no active IdP session during the redirect, the user is prompted to sign in to the IdP and then use SSO to sign in to the app for a new session.
+
+#### Non-privileged access use case
+
+For non-privileged access apps, such as a loyalty program catalog, sessions can be active for a long period of time (even days) and app developers don't need to implement session timeouts to restrict access. This is especially true for mobile apps where you want your users to have a seamless and frictionless user experience. In this scenario, the application session can outlive the IdP session. Application sessions can still track user interactions without the IdP session. However, when the user signs out and ends the application session, without the IdP session, the user is redirected to the IdP sign-in page and must authenticate with the IdP to gain access to the app, as before.
+
+### Terminate sessions
+
+#### Local logout
+
+Local logout is the act of signing the user out of the app and terminating the application session. The user IdP session is still active and other application sessions aren't affected by terminating a specific local application session. Local logout is managed by the app. If the user wants to access the app again, they are automatically authenticated through the active IdP session into the app and the app creates a new session for the user.
+
+See [Sign users out of your app](/docs/guides/sign-users-out/-/sign-out-of-your-app/) for an example of a Local logout implementation with Okta.
+
+#### Single logout
+
+Single logout is the act of signing the user out of the Okta IdP and ending the IdP session. Typically, the user is redirected to re-authenticate into Okta to start a new IdP session. Active application sessions for the user can still persist depending on the restrictive nature of the app. For non-privileged access apps, the user can still access the app within the scope of the application session. For example, a user can still browse through a catalog of vacation experiences and add to their cart. When this user is ready to check out and make a payment, the app can enforce a privileged access workflow and redirect the user to re-authenticate with the IdP to secure an IdP session as well as an application session.
+
+Apps within the Okta org can trigger a Single logout. See [Single Logout in applications](https://help.okta.com/en/prod/Content/Topics/Apps/Apps_Single_Logout.htm).
```

Comment:
is it Single logout, Single Logout, or single logout?

Reply:
Thanks - updating to "single logout"